src/mm-mapp.js

```diff
@@ -72,22 +72,40 @@ export class MmMapp extends HTMLElement {
   constructor() {
     // Always call super first in constructor
     super();
-    // SUPER IMPORTANT TO SET THIS UP FIRST SO THAT LEAFLET ISN'T WORKING WITH
```

Comment:
Hi Robert, Can you explain why this isn't as important as I thought?  It points to a fundamental lack of knowledge on my part I think.

Reply:
This is the same as https://github.com/Maps4HTML/Web-Map-Custom-Element/pull/108#discussion_r453887570.